display-property-flex/flexblocks.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+
+<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Blocks using FLEX</title>
+    <link rel="stylesheet" href="flexblocks_styles.css">
+</head>
+
+<body>
+<div class="main-box">
+    <div class="block pale">1</div>
+    <div class="block green">2</div>
+    <div class="block pink">3</div>

```

Comment:
Blocks 1, 2 and 3 should be in row wrapper. We will discuss this, but for this case it is not correct